src/data-types/wrappers.h

```diff
@@ -0,0 +1,326 @@
+#pragma once
+
+#include <errno.h>
+#include <stdarg.h>
+
+#ifdef EOF // stdio.h
```

Comment:
On Thu, Mar 05, 2015 at 01:41:10PM -0800, Hoà V. DINH wrote:
> > @@ -0,0 +1,326 @@
> > +#pragma once
> > +
> > +#include <errno.h>
> > +#include <stdarg.h>
> > +
> > +#ifdef EOF // stdio.h
> Maybe you could just include <stdio.h> (and same for other locations)?
> It would make this file simpler.

I did. Could you help me please with the issue that src/engine seems to
be treated different than all other parts relating to include paths?

Yours,
VB.
-- 
Volker Birk
Oberer Graben 4, 8400 Winterthur, Schweiz
mailto:vb@dingens.org  http://fdik.org